src/jttt/Core/AlphaBetaStrategy.java

```diff
@@ -0,0 +1,112 @@
+package jttt.Core;
+
+import java.util.HashMap;
+
+import static java.lang.Math.max;
+import static java.lang.Math.min;
+
+public class AlphaBetaStrategy implements AIMoveStrategy {
+    private static int DISPLAY_POSITION_OFFSET = 1;
+    private static final int SCORE_FOR_DRAW = 0;
+    private static Integer DEPTH = 0;
+    private static int INITIAL_SCORE = 1000000;
+    private static int INITIAL_ALPHA = -10000000;
+    private static int INITIAL_BETA = 10000000;
+    private static String MOVE_KEY = "Move";
+    private static String SCORE_KEY = "Score";
+    private Counter counter;
+
+    public AlphaBetaStrategy() {
+    }
+
+    public int calculateNextMove(Board board, Counter counter) {
+        this.counter = counter;
+        DEPTH = board.numberOfOpenPositions();
+        return indexToDisplayPosition(aBMinimax(DEPTH, this.counter, INITIAL_ALPHA, INITIAL_BETA, board));
+    }
+
+    private HashMap<String, Integer> aBMinimax(int depth, Counter currentCounter, int alpha, int beta, Board currentBoard) {
```

Comment:
You can simply return a `Map<String, Integer>` to not reveal which implementation you use.